curriculum/challenges/english/25-front-end-development/lecture-working-with-css-grid/6732268d05c3661d32a0fee8.md

```diff
@@ -14,20 +14,133 @@ Watch the lecture video and answer the questions below.
 
 ## --text--
 
-How Can You Repeat Track Listings in a Grid Layout? question?
+Which of the following is the correct syntax for repeating track listings?
```

Comment:
Looks like you came up with a new question here rather than what's in the lecture script?

Reply:
Yeah, I changed a couple of things because I thought this change was better. 👍